config/passport.js

```diff
@@ -1,28 +1,65 @@
-import passport from "passport";
-import strategies from "passport-local";
-import models from '../models/';
+import passport from 'passport';
+import strategies from 'passport-local';
+import { Strategy as JWTStrategy, ExtractJwt } from 'passport-jwt';
+import dotenv from 'dotenv';
+import models from '../models';
+
+dotenv.config();
 
 const LocalStrategy = strategies.Strategy;
-const User = models.User;
+const { User } = models;
 
-passport.use(
+passport.use('signup',
   new LocalStrategy(
     {
-      usernameField: "user[email]",
-      passwordField: "user[password]"
+      usernameField: User.email,
+      passwordField: User.password
     },
-    function(email, password, done) {
-      User.findOne({ email: email })
-        .then(function(user) {
-          if (!user || !user.validPassword(password)) {
-            return done(null, false, {
-              errors: { "email or password": "is invalid" }
-            });
-          }
+    async (email, password, done) => {
+      try {
+        const user = await User.findOne({ where: { email: email } });
+        if(!email || !password) {
+          return done(null, false, { message: 'The email or password has not to be empty' });
+        }
+        if(user) {
+          return done(null, false, { message: 'The email is already taken' });
```

Comment:
It should sound good to return that the email is used or saying that the account exists instead of saying that the email has taken because here we are requesting the user to use their existing email not to create an email for them.